app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -222,7 +220,8 @@ private static LoyaltyCard updateTempState(LoyaltyCard loyaltyCard, LoyaltyCardF
                 (int) (fieldName == LoyaltyCardField.starStatus ? value : loyaltyCard.starStatus),
                 0, // Unimportant, always set to null in doSave so the DB updates it to the current timestamp
                 100, // Unimportant, not updated in doSave, defaults to 100 for new cards
-                (int) (fieldName == LoyaltyCardField.archiveStatus ? value : loyaltyCard.archiveStatus)
+                (int) (fieldName == LoyaltyCardField.archiveStatus ? value : loyaltyCard.archiveStatus),
+                (int) (fieldName == LoyaltyCardField.usage_number ? value : 0)
```

Comment:
This should use the variable, I think otherwise the usage will be reset to 0 every time you edit the card.

Also, please use camel-case like everywhere else.